src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -5552,6 +5565,21 @@ private ServiceSpec createServiceSpec(final String serviceName,
         .build();
   }
 
+  private ServiceSpec createBadServiceSpec(final String serviceName) {
```

Comment:
Do you need this bit of code to be in a method rather than just part of the test? Do you plan to re-use it?